pydreamscreen/network/__init__.py

```diff
@@ -0,0 +1,14 @@
+"""Network facilitator - allows discovering devices in multi-interface settings."""
+import logging
+
+from .discover import (get_networks, get_broadcasts)
+
+__version__ = '0.0.7'
```

Comment:
Make versioning consistent? 

```suggestion
__version__ = '0.0.9'
```